src/Ethereum/EIP4337.cpp

```diff
@@ -7,10 +7,9 @@
 #include "ABI.h"
 #include "AddressChecksum.h"
 #include "EIP1014.h"
-#include "EIP1967.h"
 #include "Hash.h"
 #include "HexCoding.h"
-#include <iostream>
+#include <Webauthn.h>
```

Comment:
this file is missing?

Reply:
Yeah, cause we need to merge https://github.com/trustwallet/wallet-core/pull/3145 first